ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/AbstractJsonRpcHttpBySpecTest.java

```diff
@@ -281,4 +285,9 @@ private String filterStringTrace(final String expectedResult) {
     final Matcher m = GAS_MATCH_FOR_TRACE.matcher(expectedResult);
     return m.replaceAll("");
   }
+
+  private String filterRevertReason(final String expectedResult) {
```

Comment:
This does not properly test the revert reason.  Our code could now output random corrupted bytes for the revert reason and it would pass the tests.

My thinking is we add the revert reasons to the test vectors to ensure we have stable and expected output for the reverts.  Since OE is deprecated and being end-of-lifed at the merge we don't need to worry about re-generating test vectors against OE.

Reply:
fair point. (although the generation of revertReason itself is tested in other places in the code). I've modified 4 test files and added a comment to the JSON.